inst/app/data/translation.json

```diff
@@ -0,0 +1,169 @@
+{
+  "cultural_date_format": "%d-%m-%Y",
+  
+  "languages": ["en", "zh"],
+  
+  "translation": [
+    
+    {
+      "en": "Terms of Use",
+      "zh": "使用條款"
+    },
+    
+    {
+      "en": "Please select a language",
+      "zh": "請選擇一種語言"
```

Comment:
I Google translated this...is it correct?

Reply:
Yes correct!